Antarctic blog post (#336)
ropensci/roweb2: content/blog/2018-11-13-antarctic.md

---
slug: "antarctic"
title: The Antarctic/Southern Ocean rOpenSci community
authors:

>   - name: Ben Raymond and Michael Sumner on behalf of the community

Reviewer: ```suggestion
  - name: [Ben Raymond](https://github.com/raymondben)
  - name: [Michael Sumner](https://github.com/mdsumner)
    twitter: mdsumner
  - name: on behalf of the community
```

date: 2018-11-13
categories: blog

> topicid: 1234

Reviewer: ```suggestion
topicid: 
twitterImg: img/blog-images/2018-11-13-antarctic/plotanim-1.gif
```

Reviewer: I will add the `topicid` at time of publication.

`twitterImg` is set so when the link to this blog post is included in a tweet, that image will be associated with the tweet.

``` r
plot(bx)
lines(ele$lon, ele$lat)
```

<img src="/img/blog-images/2018-11-13-antarctic/plotbathy-1.png" style="display: block; margin: auto;" />

We can also extract the depth values along our track using the `extract()` function in `raadtools`:

``` r
ele$depth <- extract(readtopo, ele[, c("lon", "lat")], topo = "etopo2")
```

Plot the histogram of depth values, showing that most of the track points are located in relatively shallow waters:

``` r
ggplot(ele, aes(depth)) + geom_histogram(bins = 100) + theme_bw()
```

<img src="/img/blog-images/2018-11-13-antarctic/histbathy-1.png" style="display: block; margin: auto;" />

This type of extraction will also work with time-varying data — for example, we can extract the sea-ice conditions along our track, based on each track point's location and time:

``` r
ele$ice <- extract(readice, ele[, c("lon", "lat", "date")])
## points outside the ice grid will have missing ice values, so fill them with zeros
ele$ice[is.na(ele$ice)] <- 0
ggplot(ele, aes(date, ice, colour = lat)) + geom_path() + theme_bw()
```

<img src="/img/blog-images/2018-11-13-antarctic/temp2-1.png" style="display: block; margin: auto;" />

> Or a fancy animated plot, using `gganimate` (code not shown for brevity, but you can find it in the [page source](https://raw.githubusercontent.com/ropensci/roweb2/master/content/blog/2018-11-13-antarctic.md)). The hypnotic blue line shows the edge of the sea ice as it grows over the winter season:

Reviewer: can you add text to say what the yellow/orange moving dot is?

Author: Done. The preview build is now failing but I think that's just because there's an empty topicid in the YAML

---
slug: "antarctic"
title: The Antarctic/Southern Ocean rOpenSci community
authors:

>   - name: [Ben Raymond](https://github.com/raymondben)

Reviewer: ```suggestion
  - name: Ben Raymond
    url: https://github.com/raymondben
```

>   - name: [Michael Sumner](https://github.com/mdsumner)

Reviewer: ```suggestion
  - name:  Michael Sumner
    url: https://github.com/mdsumner
    twitter: mdsumner
```

>     twitter: mdsumner

Reviewer: ```suggestion
```
